django/db/backends/oracle/base.py

```diff
@@ -241,7 +241,8 @@ def init_connection_state(self):
         if not self.get_autocommit():
             self.commit()
 
-    def create_cursor(self, name=None):
+    def create_cursor(self, name=None, *args, **kwargs):
+        # It is not possible to pass args/kwargs in oracle cursor
```

Comment:
Then it should be documented that it isn't supported on all backends? Why does Oracle have a test if this doesn't work?